package.json

```diff
@@ -172,7 +172,8 @@
   },
   "private": true,
   "dependencies": {
-    "ua-parser-js": "^1.0.2"
+    "@stripe/stripe-js": "^1.20.3",
+    "ua-parser-js": "^1.0.1"
```

Comment:
Why this downgrade was needed?

Reply:
This got lost in some conflict resollution. reverting it.